.github/workflows/ci.yml

```diff
@@ -52,6 +52,8 @@ jobs:
       - name: make test
         run: make test
   e2e:
+    # This job does not run on forked repositories unless there is a self-runners.
```

Comment:
The comment is not right. with this configuration, the job can't run even with self-runners.

Reply:
Oh yeah, that's an inappropriate comment